src/io/flutter/actions/InstallSdkAction.java

```diff
@@ -30,6 +31,9 @@
 @SuppressWarnings("ComponentNotRegistered")
 public class InstallSdkAction extends DumbAwareAction {
 
+  //TODO(pq): add support for "git.exe"
+  private static final String GIT_EXECUTABLE = "git";
```

Comment:
FYI: I didn't flip this (in the absence of an easy way to test today) but can in a follow-up if you'd like to see it.